addons/website_sale/views/templates.xml

```diff
@@ -269,23 +269,6 @@
                 (name_desc_label, 'name desc')
             ]"/>
```

Comment:
... is all of this still useful ? Should not we keep the possibility to show the sort or not ?

Reply:
Yes you're right we should be able to sort products on the result page. Sorry, at first I did not imagine working like this :smiley: